packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -385,6 +365,7 @@ export async function runApiExtraction({
           reportFileName: 'api-report.md',
           reportFolder: projectFolder,
           reportTempFolder: resolvePath(outputDir, '<unscopedPackageName>'),
+          includeForgottenExports: packageDir === 'plugins/git-release-manager',
```

Comment:
@Rugvip `plugins/git-release-manager/api-report.md` changes seems related to [this ](https://github.com/microsoft/rushstack/pull/3552)

but including forgotten export now bring a lot of new thing to review
 scoping here to `plugins/git-release-manager` to reduce noice, but adding this for all packages brings changes on multiple packages
```
The API Report for plugins/techdocs-addons-test-utils introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/permission-node introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/permission-common introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/kubernetes introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/jenkins introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/cost-insights introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/catalog-import introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/catalog-common introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for plugins/catalog introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for packages/test-utils introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for packages/core-components introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for packages/core-app-api introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
The API Report for packages/backend-plugin-api introduces new warnings. Please fix these warnings in order to keep the API Reports tidy.
```

Reply:
ok changed my mind here, instead of adding the `includeForgottenExports` i fixed the missing export on this packages